### PR TITLE
Promote master to production.

### PR DESCRIPTION
**Pull request by robputt**

### CODE REVIEW

**Comment by shawaj** [comment]
@marvinmarnold are you satisfied that the updated config container is working ok to promote to production?

**Review by vpetersson — APPROVED** [review]

**Comment by vpetersson** [comment]
@robputt hold this.

**Comment by shawaj** [comment]
@vpetersson @robputt not sure we should promote this to production unless we've also tested the config thoroughly?

**Comment by vpetersson** [comment]
Yeah i didn't realized we bumped `hm-config` here. Let's not roll those changes out. @robputt can you cherry pick in the changes for the hm-miner only?

**Review by vpetersson — CHANGES_REQUESTED** [review]
Let's only upgrade hm-diag.

File: docker-compose.yml [hunk 1]
```diff
@@ -27,7 +27,9 @@ services:
       - 'pktfwdr:/var/pktfwd'
 
   helium-miner:
-    image: "nebraltd/hm-miner:6aacef4c2fcab92aee5fbdcc5ac6ca5927b97421"
+    image: "nebraltd/hm-miner:b404b50721f87c0f70e60bb8f5ec0a75d05b2445"
+    expose:
+      - "4467"
```
Comment (marvinmarnold) [inline docker-compose.yml:32]: What's this for?

**Review by marvinmarnold — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (vpetersson) [reply docker-compose.yml:32]: Good catch. That's the RPC interface. That should probably only be exposed locally.

Comment (robputt) [reply docker-compose.yml:32]: This is only exposed within the Docker internal network and allows the hm-diag / hm-config containers to reach the JSON RPC API. Expose is used to expose ports internally. Ports is used to expose ports to the host / external network.

**Comment by robputt** [comment]
See https://github.com/NebraLtd/helium-miner-software/pull/128 for the Cherry Pick

**Pull request closed without merging**